.azure/pipelines/jobs/default-build.yml

```diff
@@ -115,7 +115,7 @@ jobs:
         ${{ if eq(variables['System.TeamProject'], 'internal') }}:
           name: NetCoreInternal-Pool
           # Visual Studio Enterprise - contains some stuff, like SQL Server and IIS Express, that we use for testing
-          queue: BuildPool.Server.Amd64.VS2019
+          queue: BuildPool.Windows.10.Amd64.VS2019.Pre
```

Comment:
For future reference, the Prerelease Server queues were removed to save capacity. We can \ should switch back to Server once 16.7 or newer is available in the non-PRE queues.